meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -266,225 +121,59 @@ export const ShowStyleVariantsSettings = withTranslation()(
-								<span className="settings-studio-showStyleVariants-table__drag">
-									<FontAwesomeIcon icon={faGripLines} />
-								</span>
-								{showStyleVariant.name || t('Unnamed variant')}
-							</th>
-							<td className="settings-studio-showStyleVariant__actions table-item-actions c3">
-								<button className="action-btn" onClick={() => this.downloadShowStyleVariant(showStyleVariant)}>
-									<FontAwesomeIcon icon={faDownload} />
-								</button>
-								<button className="action-btn" onClick={() => this.copyShowStyleVariant(showStyleVariant)}>
-									<FontAwesomeIcon icon={faCopy} />
-								</button>
-								<button className="action-btn" onClick={() => this.editItem(showStyleVariant._id)}>
-									<FontAwesomeIcon icon={faPencilAlt} />
-								</button>
-								<button className="action-btn" onClick={() => this.confirmRemove(showStyleVariant)}>
-									<FontAwesomeIcon icon={faTrash} />
-								</button>
-							</td>
-						</tr>
-					</tbody>
-					<tbody>
-						{this.isItemEdited(showStyleVariant._id) && (
-							<tr className="expando-details hl">
-								<td colSpan={5}>
-									<div>
-										<div className="mod mvs mhs">
-											<label className="field">
-												{t('Name')}
-												<EditAttribute
-													modifiedClassName="bghl"
-													attribute={'name'}
-													obj={showStyleVariant}
-													type="text"
-													collection={ShowStyleVariants}
-													className="input text-input input-l"
-												></EditAttribute>
-											</label>
-										</div>
-									</div>
-									<div className="row">
-										<div className="col c12 r1-c12 phs">
-											<ConfigManifestSettings
-												t={this.props.t}
-												i18n={this.props.i18n}
-												tReady={this.props.tReady}
-												manifest={this.props.blueprintConfigManifest}
-												collection={ShowStyleVariants}
-												configPath={'blueprintConfig'}
-												alternateObject={this.props.showStyleBase}
-												object={showStyleVariant}
-												layerMappings={this.props.layerMappings}
-												sourceLayers={this.props.sourceLayers}
-												subPanel={true}
-											/>
-										</div>
-									</div>
-									<div className="mod alright">
-										<button className="btn btn-primary" onClick={() => this.finishEditItem(showStyleVariant._id)}>
-											<FontAwesomeIcon icon={faCheck} />
-										</button>
-									</div>
-								</td>
-							</tr>
-						)}
-					</tbody>
-				</React.Fragment>
-			)
-		}
-
-		VariantList = ({ children, className }) => {
-			const [, drop] = useDrop({
-				accept: ShowStyleDragDropTypes.VARIANT,
-				drop: () => this.persistStateVariants(),
-				collect: (monitor: DropTargetMonitor) => ({
-					isOver: monitor.isOver(),
-					canDrop: monitor.canDrop(),
-				}),
-			})
-
+		private provideShowStyleVariantItem = (variant: ShowStyleVariant, index: number) => {
```

Comment:
Just call it `renderShowStyleVariantItem` or even `renderShowStyleVariant`

Reply:
Changed!